FindTown/FindTown/Presentation/MapScene/Map/SubViews/MapDetailCategoryView.swift

```diff
@@ -5,8 +5,8 @@
 //  Created by 장선영 on 2023/01/02.
 //
 
-import Foundation
 import UIKit
+import FindTownUI
 
 public class MapDetailCategoryView: UIView {
```

Comment:
```suggestion
final class MapDetailCategoryView: UIView {
```

이거 아마 빼오시면서 public 삭제하는거 까먹으신 거 같아요..!! 
MapDetailCategoryView, MapDetailComponentView 2개입니다~~

Reply:
오 맞네요!! 수정하겠습니다~!